bgpd/bgp_damp.c

```diff
@@ -557,6 +558,13 @@ void bgp_damp_info_clean(struct bgp_damp_config *bdc, afi_t afi, safi_t safi)
 		list = &bdc->reuse_list[i];
 		while ((rn = SLIST_FIRST(list)) != NULL) {
 			bdi = rn->info;
+			if (bdi->lastrecord == BGP_RECORD_UPDATE) {
+				bgp_aggregate_increment(bgp, &bdi->dest->p,
```

Comment:
Is this valid only for UPDATEs, WITHDRAWs not involved in this clear process?

Reply:
Similar to bgp_reuse_timer(), it is used only for BGP_RECORD_UPDATE.